src/main.cpp

```diff
@@ -49,8 +49,6 @@ using namespace std;
 
 TConsole *  spDebugConsole = 0;
 
-extern Q_CORE_EXPORT int qt_ntfs_permission_lookup;
```

Comment:
Since it is never incremented (to activate the Qt Library feature) it is dead-code.  But the setting concerned (which __is__ Windows specific) may need to be used at some point to try and partially overcome the problems with the difference between the 'Doze file permission model and those of other platforms.  For reference see: [QFileDevice](http://doc.qt.io/qt-5/qfiledevice.html) documentation.